src/styles.css

```diff
@@ -0,0 +1,447 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
+
+*,
+html,
+/* body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+} */
+body {
+  /* font-family: "Lato", sans-serif; */
+  font-family: "Poppins", sans-serif;
+  background-color: #333;
+  color: white;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background: #fff;
+}
+
+header {
+  width: 100%;
+  padding-left: 1rem;
+  padding-right: 1rem;
+  margin-top: 0;
+}
+
+nav {
+  height: 70px;
+  display: flex;
+  align-items: center;
+  width: 100%;
+}
+
+.logo {
+  margin-top: 20px;
+  border: 3px solid #fff;
+  border-radius: 100%;
+  width: 70px;
+  height: 70px;
+  display: grid;
+  place-items: center;
+}
+
+.menu-wrap {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  list-style-type: none;
+  gap: 0.5rem;
+}
+
+a {
+  text-decoration: none;
+
+  /* color: #333; */
+  color: #fff;
+  font-weight: bold;
+  font-size: 25px;
+}
+
+/* main {
+  padding: 2rem 1rem;
+} */
+
+.shows-container {
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(150px, 1fr));
+  grid-column-gap: 1.5rem;
+  grid-row-gap: 1.5rem;
+}
+
+.show {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+  align-items: center;
+  width: 150px;
+  padding-top: 2px;
+  padding-bottom: 2px;
+  -webkit-box-shadow: 0 0 10px rgba(0, 0, 0, 0.15);
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.15);
+}
+
+.img-wrap {
+  width: 85%;
+  height: 100px;
+}
+
+.show-img {
+  max-width: 100%;
+  height: 100%;
```

Comment:
[OPTIONAL] I think it would be better if you use the height of the images as auto. Currently, everybody on the photos looks fat, it looks like a little bit distorted. You can this repo how they handle this issue: https://kwentsir.github.io/Homeless-Box-Office--Capstone/#
<img width="859" alt="Screenshot 2023-03-24 at 9 14 27 PM" src="https://user-images.githubusercontent.com/25887253/227608760-50cda061-e47e-4f39-abb8-0e56b69ec6e9.png">